UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -623,6 +624,7 @@ public void CmdToggleLightSwitch(GameObject switchObj)
 		}
 		else
 		{
+			//TODO: Output name of player attempting to interact with light switch.
 			Logger.LogWarning("player attempted to interact with light switch through wall," +
```

Comment:
This log message doesn't have a category

Reply:
Do you think Category.NetMessage would be right for this?

Reply:
I think, for questions like these, the discord server can give you answers better and faster.